sprimber-engine/src/main/java/com/griddynamics/qa/sprimber/lifecycle/allure/AllureSprimber.java

```diff
@@ -99,7 +102,7 @@ public void testCaseStarted(TestCaseStartedEvent startedEvent) {
                 .withName(startedEvent.getTestCase().getName())
                 .withLinks(helperInfoBuilder.getLinks())
                 .withLabels(helperInfoBuilder.getLabels())
-                .withDescription(startedEvent.getTestCase().getDescription());
+                .withDescription(startedEvent.getTestCase().getParent().getDescription());
```

Comment:
Why do you use parent description here? From my point of view test result should contain scenario description or both scenario and feature descriptions, but not only feature description